sites/akvo-flow-web/src/component/Submit.js

```diff
@@ -142,6 +142,7 @@ class Submit extends Component {
         })
         .then((res) => {
           console.log(res);
+          window.parent.postMessage(`akvo.webform.saved:${saveData.form_instance_id}:${saveData.form_id}`, '*');
```

Comment:
I would suggest to use `JSON.stringify()` and send a *data structure* instead of making a new a new *language* that we need to also parse via `split(":")`